File: Python/card.py
from payment import Payment


class Card(Payment):
    cardNumber = int
    ccv = str
    dateCard = str

    def __init__(self, id, cardNumber, ccv, dateCard):
        super().__init__(id)
        self.cardNumber = cardNumber
        self.ccv = ccv 
        self.dateCard = dateCard

File: Java/Card.java
package Java;

import java.sql.Date;

public class Card extends Payment{

    Integer cardNumber;
    String ccv;
    Date dateCard;


    public Card(Integer id, Integer cardNumber, String ccv, Date dateCard) {
        super(id);   
        this.cardNumber = cardNumber;
        this.ccv = ccv;
        this.dateCard = dateCard;

    }
}
